include/psa/crypto_adjust_config_dependencies.h

```diff
@@ -48,4 +48,12 @@
 #define PSA_WANT_ALG_CMAC 1
 #endif
 
```

Comment:
Should we add something to `all.sh` that would be a non-regression test?

Reply:
I tried to think about how to test this, but I really can't figure out a possible test setup that can test this fix. Any suggestion is really welcome!